GetJobber/atlantis
# chore(components): Deprecate Toast Actions

## Motivations


Toast actions aren't being used and therefore should be removed.


### Deprecated
- `action` and `actionLabel` props on `Toast`

## Testing

1.  Install the pre-release 
2. Use Toast component and pass in the `action` and `actionLabel` props 
3. Verify that you see the deprecation warning in the your editor (note that it will only show in a tooltip and you won't actually be able to see the strikethrough due to [this typescript issue](https://github.com/microsoft/TypeScript/issues/39374))
4. Make sure you can see the deprecation comment in storybook as shown below 
![image](https://github.com/GetJobber/atlantis/assets/9436525/51b12777-d9e9-41fc-9161-bcf357caadb6)


---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
